readthedocs/projects/models.py

```diff
@@ -196,7 +196,7 @@ class Project(models.Model):
         _('Python Interpreter'),
         max_length=20,
         choices=constants.PYTHON_CHOICES,
-        default='python',
+        default='python3',
         help_text=_('(Beta) The Python interpreter used to create the virtual '
```

Comment:
Can I remove the help text about being a beta feature?

Reply:
Oh yeah, this copy should definitely change :)